src/migrations/1733808052981-item-tags-to-tag.ts

```diff
@@ -0,0 +1,134 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class Migrations1733808052981 implements MigrationInterface {
+  name = 'item-tags-to-tag-1733808052981';
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    // insert distinct tags
+    await queryRunner.query(`
+        insert into tag (name, category)
+        select DISTINCT(
+            trim(
+                -- replace faulty breaking lines into spaces
+                replace(
+                    unnest(
+                        -- break comma-separated tags into multiple tags 
+                        string_to_array(
+                            -- remove resulting quotes (") 
+                            substr(
+                                jsonb_array_elements(settings::jsonb->'tags')::text,
+                                2,
+                                length(jsonb_array_elements( settings::jsonb->'tags' )::text) - 2
+                            ),','
+                        )
+                    ),'\n',' '
+                )
+            )
+        ), 'discipline'::tag_category_enum from item
```

Comment:
This means current tags are inserted as `disciplines` ? I am ok with that, just wanted to highlight this change.

Reply:
Yes.